packages/components/.storybook/preview-head.html

```diff
@@ -1,12 +1,6 @@
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
 <!-- Storybook should have visible overflow since we have Action Logger -->
 <style>
-	body {
-		overflow-y: auto !important;
-		max-height: none !important;
-		min-height: 100vh;
```

Comment:
Seems to do nothing (perhaps with the upgrade of storybook).
Even worse, it introduce a double scroll

Reply:
You could have commit that in an other commit, even in an another PR :s
Same for the package.json on this commit